src/components/BlockingViews/BlockingView.tsx

```diff
@@ -12,10 +14,29 @@ import Navigation from '@libs/Navigation/Navigation';
 import variables from '@styles/variables';
 import type {TranslationPaths} from '@src/languages/types';
 
-type BlockingViewProps = {
-    /** Expensicon for the page */
-    icon: React.FC<SvgProps> | ImageSourcePropType;
+type RequiredIllustrationProps =
```

Comment:
@tienifr can you clarify the expected behavior you're trying to convey with these types?

It _looks_ like you're trying to say something like "either icon or animation is required, but not both". Is that right?

Maybe what you want is more like this? ([source](https://github.com/sindresorhus/type-fest/blob/main/source/merge-exclusive.d.ts))

```ts
import type {MergeExclusive} from 'type-fest';

/**
 * This page requires either an icon or an animation, but not both
 */
type RequiredIllustrationProps = MergeExclusive<
    {
        /** Expensicon for the page */
        icon: React.FC<SvgProps> | ImageSourcePropType;
    },
    {
        /** Animation for the page */
        animation: DotLottieAnimation;
    }
>;
```